packages/common/logging/__init__.py

```
@@ -0,0 +1,133 @@
+    Returns:
+        logging.Logger: Configured logger instance.
+
+    Example:
+        >>> logger = get_logger(__name__)
+        >>> logger.info("Processing document", doc_id="abc123")
+    """
+    return logging.getLogger(name)
+
+
+# Export public API
+__all__ = ["setup_logging", "get_logger", "CorrelationIdFilter", "CustomJsonFormatter"]
```

Ruff (0.14.1)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)